recipe/build.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/sh
+set -ex
 
 ./bootstrap \
+             --verbose \
              --prefix="${PREFIX}" \
              --system-libs \
+             --system-libuv \
              --no-qt-gui \
              --no-system-libarchive \
              --no-system-jsoncpp \
              --parallel=${CPU_COUNT} \
              -- \
              -DCMAKE_BUILD_TYPE:STRING=Release \
              -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
-             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib"
+             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib" \
+             -DCURSES_LIBRARY="${PREFIX}/lib/libncurses.so" \
```

Comment:
I don't think you need this. Besides this would be `.dylib` on macosx

Reply:
I'll try removing it/